engine/render/src/render/font_renderer.cpp

```diff
@@ -562,6 +563,36 @@ namespace dmRender
         te.m_SourceBlendFactor = params.m_SourceBlendFactor;
         te.m_DestinationBlendFactor = params.m_DestinationBlendFactor;
 
+        TextMetrics metrics;
+        uint32_t line_count = GetTextMetrics(font_map, params.m_Text, params.m_Width, params.m_LineBreak, params.m_Leading, params.m_Tracking, &metrics);
```

Comment:
No need to add "line_count" here if we're not using it?
It will only add a warning when compiling.

Reply:
@JCash wdym ? line_count is used below in OffsetY().

Reply:
No you're right, it was my search that failed for some reason. Please ignore this comment.